lib/airbrake/rake/tasks.rb

```diff
@@ -40,6 +41,15 @@
 
 The test exception was sent. Find it here: #{response['url']}
 OUTPUT
+    else
```

Comment:
Yes. Also if you have filters that ignore notices such as this one.
I agree, it's not crystal clear, but I like the simplicity of the current code.